ADRs/RH/SIG-SRE/ADR-00005 SLO Lifecycle.md

```diff
@@ -0,0 +1,131 @@
+
+The owner will likely be an engineer on the service's team who may work in concert with an appropriate SRE IC to affect changes to the service and related ecosystem.
+
+By way of example, some of these changes may include (but not limited to):
+
+* SLI (Service Level Indicator) instrumentation in the service
+  * Software changes
+* SLO Dashboard Changes
+  * Creation of SLO Dashboard
+  * Changes to existing SLO dashboard to incorporate new changes
+* Changes to team policies and practices to support the [Iteration Phase](#iteration-phase)
+
+### Iteration Phase
+
+Once the SLOs are adopted, the [service stakeholders][raci-chart] will enter the lifecycle loop where the team and service come to live with the SLO. Now the team uses the previous work to instrument and monitor the service. Sprint planning is influenced by the error budget. Existing SLOs are continually evaluated for needed changes, or even their retirement. Changes will want to kick off a new Research Phase to propose and understand the scope of changes.
+
+#### SLO Review
+
+Part of healthy SLO adoption is the periodic review of the service's SLO needs. These review tasks may include:
+
+* Do the service's SLOs still measure the right thing(s) to measure customer happiness and service reliability?
+* Are the SLOs' SLIs still appropriate? Are there newer, better (ex. less computationally expensive) indicators that should be used?
+* Is a SLO still needed? (ex. Does the service landscape or user experience still demand it? Has another SLO superseded it?)
+
+These steps can be used to evaluate the SLOs themselves, but there's also review on the targets set by the service's SLO(s):
+
+* Are the service's SLOs being met?
+  * If they're not, what are common reasons why not?
+* What are the common reason(s) for risks and observed hits to the error budget?
+  * How can they be mitigated in the future?
+* What is the error budget burn rate? Will corrective action be needed at the current rate?
+
+Remember that some of these review steps can be outwardly facing to include external stakeholders such as service consumers within the organization.
+
+#### Sprint Planning Hooks
+
+A primary goal of Service Level Objectives is to inform the natural cycle of work for product feature teams by adding an additional signal for use in planning work. The new signal is the service's remaining Error Budget.
```

Comment:
I feel like this sentence is so important that we should either move it to the top of this doc, or copy it to the top of the doc and keep it here, too.